dev/integration_tests/ui/lib/main.dart

```diff
@@ -4,4 +4,4 @@
 
 import 'package:flutter/widgets.dart';
 
-void main() => runApp(const Center(child: const Text('flutter run -t xxx.dart')));
+void main() => runApp(const Center(child: const Text('flutter drive lib/xxx.dart')));
```

Comment:
As far as I can tell, this is not the case. I ran it with "flutter drive --route '/smuggle-it' lib/route.dart" and it worked.